youtube_dl/extractor/nuevo.py

```diff
@@ -0,0 +1,33 @@
+# encoding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+from ..utils import float_or_none
+
+class NuevoBaseIE(InfoExtractor):
+    def _extract_nuevo(self, config_url, video_id, info=None, ignore_hd=False):
```

Comment:
hd should be always tried to be extracted whether it's present or not. There is no need in this flag.